electron/main.js

```diff
@@ -129,7 +129,19 @@ app.whenReady().then(() => {
   ipcMain.handle("run_back", async (event, ...args) => {
     const port = await getAvailablePort(args[0]);
     console.log("BACK PORT", port);
-    const command = path.join(resource_path(), executable_name("vease-back"));
+    let command;
+    if (process.env.NODE_ENV === "development") {
```

Comment:
Fait pt un if juste sur the path directory, comme ça tu mutualises la partie création de la command avec le path join et executable_name